app/models/image.rb

```diff
@@ -6,8 +6,8 @@ class Image < ActiveRecord::Base
   #has_many :comments, :dependent => :destroy
   #has_many :likes, :dependent => :destroy
   #has_many :tags, :dependent => :destroy
-  has_one :user, :foreign_key => :uid
-  has_one :node, :foreign_key => :nid
+  belongs_to :user, :foreign_key => :uid
+  belongs_to :node, :foreign_key => :nid
```

Comment:
Made this change so that **images** fixture could be associated to **node** fixture. Also it makes more sense this way as a **node** and an **user** `has_many :images`. So **images** must `belongs_to `them.